src/main.rs

```diff
@@ -21,37 +19,21 @@ extern crate vorbis;
 extern crate time;
 extern crate tempfile;
```

Comment:
Yupp. Done. I left in the print_track function in case we need it for debugging and to keep the diff more atomic.